browser/src/VariantList/VariantFlag.tsx

```diff
@@ -47,18 +60,21 @@ export const FLAGS_CONFIG = {
 }
 
 type Props = {
-  type: any // TODO: PropTypes.oneOf(Object.keys(FLAGS_CONFIG))
+  type: string
   variant: any
 }
 
 const VariantFlag = ({ type, variant }: Props) => {
-  // @ts-expect-error TS(7053) FIXME: Element implicitly has an 'any' type because expre... Remove this comment to see the full error message
-  const { label, level, formatTooltip } = FLAGS_CONFIG[type]
-  return (
-    <Badge level={level} tooltip={formatTooltip(variant)}>
-      {label}
-    </Badge>
-  )
+  if (type in FLAGS_CONFIG) {
```

Comment:
hypothetically this check shouldn't be necessary but given the bug that got us looking at this in the first place, a little defensive coding seems appropriate here

Reply:
Yeah. Figure if it broke those pages once, nip it in the bud in case something slips through the cracks in the future.